app/components/companion_windows_component.rb

```diff
@@ -63,6 +63,7 @@ def outer_container_data
         data[:fullscreen_target] = 'area'
         data[:fullscreen_close_value] = 'Exit full screen'
       end
+      data[:companion_window_auto_expand_value] = render_content_list_panel?
     end
```

Comment:
Sorry...one more thing. Couldn't this new attribute go in the initial hash definition under `iiif_manifest_loader_iiif_manifest_value: iiif_v3_manifest_url` as `companion_window_auto_expand_value: render_content_list_panel?`?

Reply:
@corylown done.